.github/workflows/build.yml

```diff
@@ -427,6 +427,57 @@ jobs:
             library-x86_64-linux-android
           failOnError: false
 
+  alpha-release:
+    name: Alpha Release
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    permissions:
+      id-token: write
+    runs-on: ubuntu-latest
+    env:
+      NPM_TAG: 'alpha'
+    needs: 
+      - create-ios-xcframework
+      - create-android-library
+    defaults:
+      run:
+        working-directory: ./wrappers/javascript
+    timeout-minutes: 7
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+          cache: "yarn"
+          cache-dependency-path: "wrappers/javascript"
+      - name: install
+        run: yarn install --immutable
+
+      - name: Fetch Android libraries
+        uses: actions/download-artifact@v4
+        with:
+          name: android-libraries
+          path: indy-vdr-react-native/native/mobile/android/
+
+      - name: Fetch iOS Framework
+        uses: actions/download-artifact@v4
+        with:
+          name: indy_vdr.xcframework
+          path: indy-vdr-react-native/native/mobile/ios/
+
+      # On push to main, release unstable version
+      - name: Release alpha
+        run: |
+          git update-index --assume-unchanged $(find . -type d -name node_modules -prune -o -name 'package.json' -print | tr "\n" " ")
+          #export NEXT_VERSION_BUMP=$(yarn next-version-bump)
+          export NEXT_VERSION_BUMP="major"
```

Comment:
This will always increase the version by major, not sure if that's what we want?

Maybe we can just do the current version + alpha-{commit-sha}?